server/bin/org_migrator.py

```diff
@@ -778,45 +798,41 @@ def parse_options():
     parser = OptionParser(usage=usage)
 
     parser.add_option("--debug", action="store_true", default=False,
-            help="Enables debug output")
+        help="Enables debug output")
 
     parser.add_option("--dbtype", action="store", default='postgresql',
-            help="The type of database to target. Can target MySQL, MariaDB or PostgreSQL; defaults to PostgreSQL")
+        help="The type of database to target. Can target MySQL, MariaDB or PostgreSQL; defaults to PostgreSQL")
     parser.add_option("--username", action="store", default='candlepin',
-            help="The username to use when connecting to the database; defaults to 'candlepin'")
+        help="The username to use when connecting to the database; defaults to 'candlepin'")
     parser.add_option("--password", action="store", default='',
-            help="The password to use when connecting to the database")
+        help="The password to use when connecting to the database")
     parser.add_option("--host", action="store", default='localhost',
-            help="The hostname/address of the database server; defaults to 'localhost'")
+        help="The hostname/address of the database server; defaults to 'localhost'")
     parser.add_option("--port", action="store", default=None,
-            help="The port to use when connecting to the database server")
+        help="The port to use when connecting to the database server")
     parser.add_option("--db", action="store", default='candlepin',
-            help="The database to use; defaults to 'candlepin'")
+        help="The database to use; defaults to 'candlepin'")
     parser.add_option("--file", action="store", default='export.zip',
-            help="The name of the file to export to or import from; defaults to 'export.zip''")
+        help="The name of the file to export to or import from; defaults to 'export.zip''")
```

Comment:
There's an extra ' (single bracket) at the end of the sentence

Reply:
Fixed